task/02-numbers-tasks.js

```diff
@@ -89,10 +91,10 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+  if (b === 0 || !b) return 0;
+  return (-1 * b) / a;
```

Comment:
`if` statement is redundant in this case, we can just simplify this function to `return -b / a`